lib/acl-checker.js

```diff
@@ -144,6 +146,49 @@ class ACLChecker {
   isAcl (resource) {
     return resource.endsWith(this.suffix)
   }
+
+  static createFromLDPAndRequest (resource, ldp, req) {
```

Comment:
This could be moved as a method to `ldp.js` instead, something like `createACLFromRequest(resource, req)` if that's a better abstraction? @kjetilk @rubensworks

Reply:
I think it's fine to keep it here, but not strong opinion.